src/testapp/sessions/vehicles_machines.cpp

```diff
@@ -77,7 +77,10 @@ Session setup_mach_rocket(
         .sync_with  ({tgParts.machIds(Ready), tgParts.machUpdExtIn(New)})
         .push_to    (out.m_tasks)
         .args       ({idScnParts, idUpdMach})
-        .func_raw   (gen_allocate_mach<gc_mtMagicRocket>());
+        .func       ([] (ACtxParts& rScnParts, MachineUpdater& rUpdMach)
+    {
+        rUpdMach.localDirty[gc_mtMagicRocket].ints().resize(rScnParts.machines.perType[gc_mtMagicRocket].localIds.vec().capacity());
```

Comment:
could you elaborate on this change?

Reply:
Was to fix a compile error on windows. Now in main.